src/device-registry/models/Site.js

```diff
@@ -1,3 +1,4 @@
+require("module-alias/register");
```

Comment:
What was the motivation for adding the "alias" module to this Models file? Does it help out in the testing process? @BenjaminSsempala

Reply:
@Baalmart yes, I discovered it was required in order to access the other filesbecuse of the format of importing files i.e. 
`const eventUtil = require("@utils/create-event");`